holmes/toolset_config_tui.py

```
@@ -0,0 +1,884 @@
+        # Hint line
+        parts.append(("class:hint", "\n  Up/Down: navigate | Enter: edit/select | Ctrl+D: delete entry | Esc: cancel edit\n"))
+        return parts
+
+    # ── key bindings ──
+
+    kb = KeyBindings()
+
+    @kb.add("up")
+    @kb.add("k")
+    @kb.add("left")
+    def _up(event: Any) -> None:
```

Ruff (0.15.2)

[warning] Unused function argument: `event`

(ARG001)